parboiled-core/src/test/scala/org/parboiled2/RunningSpec.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2009-2019 Mathias Doenitz
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.parboiled2
+
+import scala.util.{Success, Try}
+import utest._
+
+object RunningSpec extends TestSuite {
+
+  class TestParser(val input: ParserInput) extends Parser {
+    //def A                 = rule('a' ~ B ~ EOI)
+    //def B                 = rule(oneOrMore('b'))
+    //def C(n: Int)(m: Int) = rule((n - m).times('c'))
```

Comment:
In #287, I have ported lots more combinator, only `times` is missing for this test, so I'm trying to add that as well now.